sections/layer2/index.tsx

```diff
@@ -31,23 +35,20 @@ const Index: FC = () => {
 			try {
 				const provider = new providers.StaticJsonRpcProvider(OVM_RPC_URL);
 				const {
-					contracts: { Synthetix, FeePool },
+					contracts: { Synthetix },
 				} = initSynthetixJS({
 					provider,
 					useOvm: true,
 				});
 
-				const [totalSupplyBN, feePeriod] = await Promise.all([
-					Synthetix.totalSupply(),
-					FeePool.recentFeePeriods('0'),
-				]);
+				const totalSupplyBN = (await Synthetix.totalSupply()) as ethers.BigNumber;
 
-				const totalSupply = totalSupplyBN / 1e18;
-				const rewards = feePeriod.rewardsToDistribute / 1e18;
+				const totalSupply = Number(totalSupplyBN) / 1e18;
 
-				setL2APR((rewards * 52) / totalSupply);
+				setL2APR((WEEKLY_L2_REWARDS * WEEKS_IN_YEAR) / totalSupply);
 				setL2AmountSNX(totalSupply);
 			} catch (e) {
+				console.log(e);
```

Comment:
I'm putting this console.log here so we can see the errors before settings the apr/amount to `0` (on `dev` I discovered that `totalSupply` was actually failing)